package.json

```diff
@@ -43,7 +43,7 @@
     "eslint-config-brigade": "^3.2.1",
     "eslint-plugin-react": "^6.10.0",
     "in-publish": "^2.0.0",
-    "jasmine-core": "^2.1.3",
+    "jasmine-core": "^2.6.4",
```

Comment:
I saw you fixed removing the caret (`^`) in a later commit. The reason this doesn't work is because it will still grab the latest 2.x.x version.